tools/filetree_example.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 4,
```

Comment:
there might be (I am not familiar, didn't check on detail) convention to clear out execution markers in notebook before saving/committing so there is no unrelated diffs in those files

Reply:
yes i cleared it out.